data.js

```diff
@@ -812,4 +815,418 @@ exports.data = [
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "REDD.plus",
+            link: "https://www.redd.plus/",
+            keywords: ["offsetting", "reforestation", "deforestation", "private-sector", "public-sector"],
+            label: "redd-dot-plus",
+        },
+        description: {
+            content: "REDD stands for Reducing emissions from deforestation and forest degradation and is part of the UN Framework Convention on Climate Change. It's a private-public initiative, seemingly setting up a registry to allow buying credits that support protection of forest areas."
+            + " The signup link doesn't work and there are [serious concerns about REDD+](https://en.wikipedia.org/wiki/Reducing_emissions_from_deforestation_and_forest_degradation#Concerns).",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "Voluntary REDD+ Database",
+            link: "http://www.fao.org/forestry/vrd/data/",
+            keywords: ["offsetting", "reforestation", "deforestation", "public-sector"],
+            label: "redd-dot-plus",
+        },
+        description: {
+            content: "This registry visualizes REDD+ initiatives in members of the [Coalition of Rainforest Nations](https://www.rainforestcoalition.org)."
+            + " It shows which countries (without rainforests) are funding these initiatives. The data doesn't come with dates and seems a bit abandoned overall."
+            + " It's hosted by the FAO (Food and Agriculture Organization at the UN)."
+            + " Also there are [serious concerns about REDD+](https://en.wikipedia.org/wiki/Reducing_emissions_from_deforestation_and_forest_degradation#Concerns).",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "Sandbag",
+            link: "https://sandbag.org.uk/",
+            keywords: ["legislation", "think-tank", "projection", "science", "ngo"],
+            label: "sandbag",
+        },
+        description: {
+            content: "This registry visualizes REDD+ initiatives in members of the [Coalition of Rainforest Nations](https://www.rainforestcoalition.org)."
+            + " It shows which countries (without rainforests) are funding these initiatives. The data doesn't come with dates and seems a bit abandoned overall."
+            + " It's hosted by the FAO (Food and Agriculture Organization at the UN)."
+            + " Also there are [serious concerns about REDD+](https://en.wikipedia.org/wiki/Reducing_emissions_from_deforestation_and_forest_degradation#Concerns).",
+            updatedBy: "Alexander", updatedAt: "2020-02-15"
+        },
+    },
+    {
+        meta: {
+            name: "C2ES",
+            link: "https://www.c2es.org/",
+            keywords: ["legislation", "think-tank", "policy", "targets-private-sector"],
```

Comment:
also network https://www.c2es.org/our-work/carbon-capture-coalition/

Reply:
I the Carbon Capture Coalition added to `content to add`

Reply:
don't got this, sorry